src/components/shared/chart/Viewport.js

```diff
@@ -17,23 +17,50 @@ import type {
 import typeof { updateProfileSelection as UpdateProfileSelection } from '../../../actions/profile-view';
 import type { ProfileSelection } from '../../../types/actions';
 
+require('./Viewport.css');
```

Comment:
Where'd this guy come from?

Reply:
Ahah, some rebase issue :) I see it's duplicated elsewhere in the file. Thanks !